### PR TITLE
docs: Update "Get started - next steps"

### PR DESCRIPTION
**Pull request by mtmattei**
## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?
Missing Resources

## What is the new behavior?
Added resources from Beginners Guide to Uno Platform
Fixed grammar and structure issues

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: doc/articles/get-started-next-steps.md [hunk 1]
```diff
@@ -1,32 +1,129 @@
...
+   
+   [Part 3 – Dialog Component & ViewModel]( https://steven-giesel.com/blogPost/a3179d55-d5be-48ba-b570-ee7d494a8b21)
+   
+   [Part 4 – Adding Elements to the Swimlane]( https://steven-giesel.com/blogPost/2d96d970-ef11-48f4-a102-9339fc362a75)
+   
+   [Part 5 – Implementing Drag and Drop Behavior]( https://steven-giesel.com/blogPost/2c025ac6-d67f-45ec-a616-009e0285c999)
+
+**PAID RESOURCES:**
+
+**6. Pluralsight - Uno Platform Fundamentals**
```
Comment (jeromelaban) [inline doc/articles/get-started-next-steps.md:63]: It's probably best to use `##` for these sections

File: doc/articles/get-started-next-steps.md [hunk 2]
```diff
@@ -1,32 +1,129 @@
...
+
+   •	If you already have a subscription on Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno solution in Visual Studio and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform.
```
Comment (jeromelaban) [inline doc/articles/get-started-next-steps.md:65]: You may want to remove the dot, same for the other parts:
```suggestion
If you already have a subscription on Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno solution in Visual Studio and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform.
```

File: doc/articles/get-started-next-steps.md [hunk 3]
```diff
@@ -1,32 +1,129 @@
...
+
+**7. Udemy – Introduction to Uno Platform**
+
+   •	A cost-effective way to get the basics is the [Uno Platform course on Udemy]( https://www.udemy.com/course/introduction-to-uno-platform/?referralCode=C9FE308096EADFB5B661) – only about $15 USD. *If you visit the page on consecutive days, it automatically kicks in with a discount.
+   
+**8. Packt - Creating Cross-Platform C# Applications with Uno Platform**
+
+   •	[Creating Cross-Platform C# Applications with Uno Platform]( https://www.packtpub.com/product/creating-cross-platform-c-applications-with-uno-platform/9781801078498) By Matt Lacey & Marcel Alexander Wagner is a helpful guide packed with tips and practical examples to help you build applications for common scenarios. It’s available on Packt as well as Amazon.ca/.com
+   
+**ADVANCED RESOURCES:**
```
Comment (jeromelaban) [inline doc/articles/get-started-next-steps.md:86]:
```suggestion
### Advanced Resources
```

**Comment by agneszitte-nventive** [comment]
@mtmattei let me know if you need any help with the markdown syntax changes

File: doc/articles/get-started-next-steps.md [hunk 4]
```diff
@@ -1,32 +1,129 @@
...
+#### 4. Microsoft blogs on Uno Platform
 
-* A knowledge of working with GitHub and Forks, Pull and Push requests is desirable. To learn more about Forks please see official GitHub documentation. [GitHub - Forks](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks).
+- WinUI Team Blog [Building cross-platform apps with WinUI and Uno]( https://blogs.windows.com/windowsdeveloper/2021/03/11/build-cross-platform-applications-with-winui-and-uno-platform/). This tutorial walks you through building a simple same-codebase app which uses WinUI InfoBar control and runs on Windows 10, WebAssembly and Android.
+- Here are two articles that will help developers [Getting started with the Surface Duo]( https://devblogs.microsoft.com/surface-duo/tag/uno-platform/).
+   
+#### 5. Building a Todo App with Uno Platform
+
+- [Steven Giesel](https://github.com/linkdotnet). recently explored the in’s and out of Uno Platform and in turn documented his experience with a great blog series that dive’s further into What is Uno Platform and how to build a multi-platform Todo App.
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-next-steps.md:49]:
```suggestion
- [Steven Giesel](https://github.com/linkdotnet). recently explored the in’s and out of Uno Platform and in turn documented his experience with a great blog series that dives further into What is Uno Platform and how to build a multi-platform Todo App.
```

File: doc/articles/get-started-next-steps.md [hunk 5]
```diff
@@ -1,32 +1,129 @@
...
+   
+   [Part 4 – Adding Elements to the Swimlane]( https://steven-giesel.com/blogPost/2d96d970-ef11-48f4-a102-9339fc362a75)
+   
+   [Part 5 – Implementing Drag and Drop Behavior]( https://steven-giesel.com/blogPost/2c025ac6-d67f-45ec-a616-009e0285c999)
+
+## PAID RESOURCES:
+
+#### 6. Pluralsight - Uno Platform Fundamentals
+
+- If you already have a subscription on Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno solution in Visual Studio and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform.
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-next-steps.md:65]:
```suggestion
- If you already have a subscription to Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno solution in Visual Studio and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform.
```

File: doc/articles/get-started-next-steps.md [hunk 6]
```diff
@@ -1,32 +1,129 @@
...
+   
+#### 8. Packt - Creating Cross-Platform C# Applications with Uno Platform
+
+- [Creating Cross-Platform C# Applications with Uno Platform]( https://www.packtpub.com/product/creating-cross-platform-c-applications-with-uno-platform/9781801078498) By Matt Lacey & Marcel Alexander Wagner is a helpful guide packed with tips and practical examples to help you build applications for common scenarios. It’s available on Packt as well as Amazon.ca/.com
+   
+## ADVANCED RESOURCES:
+
+#### 9. Raspberry Pi
+
+- If your looking to bring your cross platform apps to Raspberry Pi with Uno Platform, Peter Gallagher wrote an [article to help you get started]( https://www.petecodes.co.uk/developing-uwp-apps-for-the-raspberry-pi-with-uno-platform/) or try our [getting started with Raspberry Pi documentation]( https://platform.uno/docs/articles/guides/raspberry-pi/raspberry-pi-intro.html).
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-next-steps.md:90]:
```suggestion
- If you are looking to bring your cross-platform apps to Raspberry Pi with Uno Platform, Peter Gallagher wrote an [article to help you get started]( https://www.petecodes.co.uk/developing-uwp-apps-for-the-raspberry-pi-with-uno-platform/) or try our [getting started with Raspberry Pi documentation]( https://platform.uno/docs/articles/guides/raspberry-pi/raspberry-pi-intro.html).
```

File: doc/articles/get-started-next-steps.md [hunk 7]
```diff
@@ -1,32 +1,129 @@
...
+
+## UNO PLATFORM RESOURCES
+
+#### Uno Team Community Support
+
+- If you have a question about Uno Platform that others might benefit from, post it on [Uno Platform GitHub discussions](https://github.com/unoplatform/uno/discussions) or If you think you've encountered a bug (e.g. something works on UWP but not on another platform), [check for an existing issue on GitHub]( https://github.com/unoplatform/uno/issues) and create one if none exists. We don't like bugs, but we do like bug reports!
+
+- For more chat-like troubleshooting, visit with the Uno Platform team and the Uno community on our [Discord Channel #uno-platform]( https://discord.gg/eBHZSKG). Both have our engineering team monitoring discussions and answering questions.
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-next-steps.md:98]:
```suggestion
- For more chat-like troubleshooting, visit the Uno Platform team and the Uno community on our [Discord Channel #uno-platform]( https://discord.gg/eBHZSKG). Both have our engineering team monitoring discussions and answering questions.
```

File: doc/articles/get-started-next-steps.md [hunk 8]
```diff
@@ -1,32 +1,129 @@
...
+
+#### Official Documentation
+
+- Our [Uno Platform Documentations]( https://platform.uno/docs/articles/intro.html) page is always a great reference point when looking for additional information. We are continuously publishing documentation covering everything from getting started, migration, components, tutorials, troubleshooting and even contributing. To find out more about cross-platform development with Uno, consult the rest of the docs here, including a list of [supported features]( https://github.com/unoplatform/uno/blob/master/doc/articles/supported-features.md) and [general development tips]( https://github.com/unoplatform/uno/blob/master/doc/articles/using-uno-ui.md).
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-next-steps.md:102]:
```suggestion
- Our [Uno Platform Documentations]( https://platform.uno/docs/articles/intro.html) page is always a great reference point when looking for additional information. We are continuously publishing documentation covering everything from getting started, migration, components, tutorials, troubleshooting, and even contributing. To find out more about cross-platform development with Uno, consult the rest of the docs here, including a list of [supported features]( https://github.com/unoplatform/uno/blob/master/doc/articles/supported-features.md) and [general development tips]( https://github.com/unoplatform/uno/blob/master/doc/articles/using-uno-ui.md).
```

File: doc/articles/get-started-next-steps.md [hunk 9]
```diff
@@ -1,32 +1,129 @@
...
+   
+#### Uno Platform YouTube
+
+- The [Uno Platform YouTube](https://www.youtube.com/c/UnoPlatform) channel is also a great source for learning. On top of publishing tutorials and demos, we also have weekly live coding sessions that tackle different areas of Uno Platform.
+   
+#### Uno Platform Blog
+
+- The [Uno Platform blog](http://platform.uno/blog) is frequently active, publishing articles weekly on a wide array of topics and varying complexity.
+Here are some highlighted articles form the Uno Blog:
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-next-steps.md:111]:
```suggestion
Here are some highlighted articles from the Uno Blog:
```

File: doc/articles/get-started-next-steps.md [hunk 10]
```diff
@@ -1,32 +1,129 @@
...
+
+- [Uno Azure DevOps (UADO)](https://github.com/unoplatform/uado) is our reference app, built from the ground up for Uno. [Uno Platform Workshop]( https://github.com/unoplatform/workshops) - In this workshop, you will learn by creating a real-world mobile & web app using Uno Platform. The workshop covers: The philosophy behind Uno development, Tooling, and component ecosystem, creating rich and responsive UI, and Interoperability with native code. And of course, we will show how to contribute to the Uno Platform itself.
+
+We're excited for you to join the Uno Platform community, if you're ready to contribute to Uno Platform, great! Check out our guides on [building Uno]( https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/debugging-uno-ui.md) and [creating UI samples]( https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md). Knowledge of working with GitHub and Forks, Pull and Push requests is desirable. 
+
+To learn more about Forks please see official GitHub documentation. [GitHub - Forks]( https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks).
```
Comment (agneszitte-nventive) [inline doc/articles/get-started-next-steps.md:129]:
```suggestion
To learn more about Forks please see the official GitHub documentation. [GitHub - Forks]( https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks).
```

**Review by agneszitte-nventive — APPROVED** [review]

**Pull request merged**